README.rst

```diff
@@ -125,7 +125,11 @@ For GitHub Actions
 
    name: CI
 
-   on: [push, pull_request]
+   on:
+    push:
+      branches: [ master ]
+    pull_request:
+      branches: [ master ]
```

Comment:
I guess -- but I'm not the maintainer here -- one of the reasons this was not done as you suggest is that it makes an assumption, namely that there is a `master` branch and that someone wouldn't want `on_push` to trigger a build in branches other than that branch.

Perhaps add some comments here to explain when (and how) this should be changed?